lib/boombox.ex

```diff
@@ -11,7 +11,7 @@ defmodule Boombox do
 
   alias Membrane.RTP
 
-  @type transcoding_policy() :: {:transcoding_policy, :always | :if_needed | :never}
+  @type transcoding_policy :: {:transcoding_policy, :always | :if_needed | :never}
```

Comment:
[NIT] I think `transcoding_policy` should be `:always | :if_needed | :never}`, `{:transcoding_policy, :always | :if_needed | :never}` is more like `transcoding_policy_opt`